src/SqlPersistence.Tests/SqlValidateStorageTypeCombinationFeatureTests.cs

```diff
@@ -36,6 +36,6 @@ public void Should_throw_when_not_valid_combination(FeatureState outboxEnabled,
         settings.Set(typeof(SqlOutboxFeature).FullName, sagaUsingSqlPersistence);
 
         var ex = Assert.Throws<Exception>(() => SqlValidateStorageTypeCombinationFeature.ValidateSagaOutboxCombo(settings));
-        Assert.That(ex.Message.Equals("Sql Persistence must be enabled for either both Sagas and Outbox, or neither.", comparisonType: StringComparison.OrdinalIgnoreCase));
+        Assert.AreEqual("When both sagas and outbox are enabled, SQL persistence must be enabled for either both sagas and outbox, or neither.", ex.Message);
```

Comment:
`Assert.That` gives a terrible failure message:

```text
Expected: True
>   But was:  False
```

`Assert.AreEqual` gives:

```text
Expected string length 77 but was 117. Strings differ at index 0.
  Expected: "Sql Persistence must be enabled for either both Sagas and Out..."
  But was:  "When both sagas and outbox are enabled, SQL persistence must ..."
  -----------^
```

As far as I can tell, the only reason to use `Assert.That` here is to have a case-insensitive comparison, but since we're copying the exception message from the production code into the test, why not copy it with the same case?

Personally I would leave out this assertion. It's not adding any value only adds the burden on updating the exception message in two places instead of one each time it's edited.